build.sbt

```diff
@@ -2,6 +2,7 @@ lazy val `sbt-pull-request-validator` = (project in file("."))
   .enablePlugins(AutomateHeaderPlugin)
 
 sbtPlugin := true
+enablePlugins(SbtPlugin)
```

Comment:
This is one of the few source breaking changes introduced in sbt over the years, specifically sbt 1.2.8.

I had to do a similar change when updating [sbt version in multi-release-jar](https://github.com/sbt/sbt-multi-release-jar/pull/26)